mqtt/aedes/mqtt.js

```diff
@@ -5,6 +5,8 @@
 
 'use strict';
 
+const version = '0.9.0';
```

Comment:
This service is written in ES 5 style. Think about using var here. If you don't want to, it's not a problem.